cardinal/server/server.go

```diff
@@ -78,13 +78,20 @@ func newSwaggerHandlerEmbed(w *ecs.World, opts ...Option) (*Handler, error) {
 		return nil, err
 	}
 
+	api.RegisterOperation("GET", "/events", runtime.OperationHandlerFunc(func(params interface{}) (interface{}, error) {
+		return struct{}{}, nil
+	}))
+
 	if err := api.Validate(); err != nil {
 		return nil, err
 	}
 
 	app := middleware.NewContext(specDoc, api, nil)
-
-	th.mux.Handle("/", app.APIHandler(nil))
+	if builder != nil {
```

Comment:
Once you remove the pointer up on line 53, the if statement here can be removed: